app/app.go

```diff
@@ -317,14 +317,21 @@ func wireCoreWorkflow(ctx context.Context, life *lifecycle.Manager, conf Config,
 	eth2Cl eth2wrap.Client, peerIDs []peer.ID, sender *p2p.Sender,
 	qbftSniffer func(*pbv1.SniffedConsensusInstance), seenPubkeys func(core.PubKey),
 ) error {
+	feeRecipientAddrs, err := lock.FeeRecipientAddresses()
+	if err != nil {
+		return err
+	}
+
 	// Convert and prep public keys and public shares
 	var (
-		corePubkeys       []core.PubKey
-		eth2Pubkeys       []eth2p0.BLSPubKey
-		pubshares         []eth2p0.BLSPubKey
-		allPubSharesByKey = make(map[core.PubKey]map[int]*bls_sig.PublicKey) // map[pubkey]map[shareIdx]pubshare
+		corePubkeys                  []core.PubKey
+		eth2Pubkeys                  []eth2p0.BLSPubKey
+		pubshares                    []eth2p0.BLSPubKey
+		allPubSharesByKey            = make(map[core.PubKey]map[int]*bls_sig.PublicKey) // map[pubkey]map[shareIdx]pubshare
+		feeRecipientAddsByEth2Pubkey = make(map[eth2p0.BLSPubKey]string)
```

Comment:
```suggestion
		feeRecipientAddrsByEth2Pubkey = make(map[eth2p0.BLSPubKey]string)
```
similarly for `feeRecipientAddsByEth2Pubkey`